### ISSUE TITLE
ArkInventoryTooltip.lua:1399: bad argument #1 to 'unpack' (table expected, got nil)

### ISSUE THREAD

**Issue by TsunamiFM** [arkayenro/arkinventory#1119]
Following error happens when holding cursor on the map for about second over the HandyNotes skull indicating rare:
```
Message: ...nterface\AddOns\ArkInventory\ArkInventoryTooltip.lua:1399: bad argument #1 to 'unpack' (table expected, got nil)
Time: Sun Sep 30 10:11:57 2018
Count: 5
Stack: ...nterface\AddOns\ArkInventory\ArkInventoryTooltip.lua:1399: bad argument #1 to 'unpack' (table expected, got nil)
[C]: in function `unpack'
...nterface\AddOns\ArkInventory\ArkInventoryTooltip.lua:1399: in function `ReloadTooltipSetGeneric'
...nterface\AddOns\ArkInventory\ArkInventoryTooltip.lua:625: in function `?'
...nterface\AddOns\ArkInventory\ArkInventoryTooltip.lua:1485: in function <...nterface\AddOns\ArkInventory\ArkInventoryTooltip.lua:1457>

Locals: (*temporary) = nil
(*temporary) = "table expected, got nil"
```

ArkInventory v3.08.14
HandyNotes v1.5.3
HandyNotes - Arathi Highlands Rares v0.4.0

**Comment by arkayenro** [comment]
must be a custom tooltip of some sort

will be fixed in the next release

**Comment by arkayenro** [comment]
fixed in 3.08.15
